src/dictionary/ui/display/WordInformationDisplay.tsx

```diff
@@ -0,0 +1,299 @@
+import React from 'react'
+import Word from 'dictionary/domain/Word'
+import withData, { WithoutData } from 'http/withData'
+import { Link, RouteComponentProps } from 'react-router-dom'
+import AppContent from 'common/AppContent'
+import { SectionCrumb, TextCrumb } from 'common/Breadcrumbs'
+import { Button, Col, Row } from 'react-bootstrap'
+import { HashLink } from 'react-router-hash-link'
+import './wordInformationDisplay.css'
+import ReactMarkdown from 'react-markdown'
+import ExternalLink from 'common/ExternalLink'
+import WordService from 'dictionary/application/WordService'
+
+const LiteratureRedirectBox = (): JSX.Element => (
+  <ExternalLink
+    style={{ color: 'black' }}
+    href="https://www.harrassowitz-verlag.de/isbn_978-3-447-04264-2.ahtml"
+    title="A Concise Dictionary of Akkadian, Harrassowitz Verlag"
+  >
+    <Row>
+      <Col xs={11} className="mx-auto">
+        <Row className="m-3 py-2 border border-dark">
+          <Col xs="auto">
+            <i className="linkContainer mt-2 fas fa-shopping-cart" />
+          </Col>
+          <Col className="linkContainer my-auto" style={{ fontSize: '0.7em' }}>
```

Comment:
Why is the class name `linkContainer`, there is no link inside`?

Use BEM naming conventions.

Do not use inline styles.